content/source/docs/registry/providers/docs.html.md

```diff
@@ -24,6 +24,18 @@ The maximum number of documents allowed for a single provider version is 1000.
 
 Each document can contain no more than 500KB of data. Documents which exceed this limit will be truncated, and a note will be displayed in the Terraform Registry.
 
+## Generating Documentation
+
+The [tfplugindocs](https://github.com/hashicorp/terraform-plugin-docs) command can be used to generate documentation for your provider in the format necessary for the Terraform Registry. It will read descriptions and schema from your code for your provider and each resource and data source and generate the relevant markdown files.
```

Comment:
```suggestion
The [tfplugindocs](https://github.com/hashicorp/terraform-plugin-docs) command can be used to automatically generate documentation for your provider in the format necessary for the Terraform Registry. It will read descriptions and schema from each resource and data source in your provider and generate the relevant markdown files for you.
```

Reply:
@paultyng is it worth mentioning (either here or in the project readme) there may be certain edge cases where resources/datasources are not properly recognized or not necessary?

Reply:
I think if there isn't one already i should just write a github issue on the repo.